internal/controllers/provisioningrequest_clusterconfig.go

```diff
@@ -81,16 +113,47 @@ func (t *provisioningRequestReconcilerTask) handleClusterPolicyConfiguration(ctx
 		return false, err
 	}
 
-	// If there are policies that are not Compliant and the configuration has not timed out,
-	// we need to requeue and see if the timeout is reached.
-	return (!allPoliciesCompliant && !allPoliciesInInform) && !policyConfigTimedOut, nil
+	// Requeue only when there's work to converge:
+	//  - present enforce policies not yet compliant and not timed out, or
+	//  - expected enforce policies not yet created and not timed out.
+	// Do not requeue for inform-only states (either present or expected).
+	requeue = ((!allPoliciesCompliant && !allPoliciesInInform) || (missingChildPolicies && !allExpectedPoliciesInInform)) && !policyConfigTimedOut
+	return requeue, nil
+}
+
+// summarizeExpectedChildPolicies summarizes if there are any expected child policies that are not yet present in the cluster,
+// and if all expected policies are inform.
+func (t *provisioningRequestReconcilerTask) summarizeExpectedChildPolicies(
+	ctx context.Context, expectedChildPolicies map[string]string, currentChildPolicies map[string]bool) (bool, bool) {
+	allExpectedPoliciesInInform := true
+	missedChildPolicyNames := []string{}
```

Comment:
nit, but I think renaming the variable from `missed` to `missing` reads better and aligns with the declaration in other places?
```suggestion
	missingChildPolicyNames := []string{}
```